src/CommunityToolkit/Xamarin.CommunityToolkit/Views/SegmentedView/Android/FormsSegments.android.cs

```diff
@@ -0,0 +1,262 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using System.Threading.Tasks;
+using Android.Content;
+using Android.Graphics;
+using Android.Graphics.Drawables;
+using Android.Util;
+using Android.Views;
+using Android.Widget;
+using AColor = Android.Graphics.Color;
+using ARes = Android.Resource;
+using AContentRes = Android.Content.Res;
+using AViews = Android.Views;
+using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.Forms.Platform.Android;
+using Xamarin.Forms;
+using Xamarin.CommunityToolkit.Extensions.Android;
+using RadioButton = Android.Widget.RadioButton;
+
+namespace Xamarin.CommunityToolkit.Android.UI.Views
+{
+	public class FormsSegments : RadioGroup, IDisposable
+	{
+		const string TAG = "FormsSegments";
+
+		readonly Context context;
+		readonly float defaultTextSize = 15.0f;
+		readonly float defaultStrokeWidth = 2.5f;
+		readonly int defaultButtonPadding = 16;
+
+		int strokeWidth;
+		bool isDisposed;
+
+		public SegmentMode DisplayMode { get; set; }
+
+		public CornerRadius CornerRadius { get; set; } = 8.0f;
+
+		public AColor TintColor { get; set; } = AColor.Rgb(14, 98, 255);
```

Comment:
Hello, this blue color is always rendering in the UI, no matter what we define in XAML.
It is intended ?

Reply:
Actually, even after I have commented this property initialization, I have realized that the bindable property is not working

Reply:
Not sure if I got it, like even if is supposed to not have anything rendered?

Reply:
I think the intended behaviour is that the value for TintColor is overwritten by the value defined in the "Color" bindable property, but it actually does not work.
Indeed, the bindable property "Color" for SegmentedControl works well in iOS, but not in Android.

Reply:
not sure about it, @hnabbasi can you answer this question?

Reply:
Hmm. That should work. I will work on this PR this week.

Reply:
@pictos, you have some time this week we can discuss an issue on Android with Asyc ?

Reply:
@hnabbasi totally, just ping me at Discord or Twitter.

The team and I thought that you hadn't time to move this forward, so I'm working in some items, so do a pull before any work (:

Reply:
Yea. I stepped away for a project. I want to finish this but had a weird issue. So put it on ice. :-)